aries-site/src/layouts/content/NotificationTag.js

```diff
@@ -0,0 +1,17 @@
+import { Tag } from 'grommet';
+
+export const NotificationTag = ({ size, color, ...rest }) => {
```

Comment:
```suggestion
export const NotificationTag = ({ size, backgroundColor, ...rest }) => {
```
this might be better naming since `color` in CSS refers to text color generally.